lib/std/builtin.zig

```diff
@@ -79,11 +79,23 @@ pub const SymbolVisibility = enum {
 /// This data structure is used by the Zig language code generation and
 /// therefore must be kept in sync with the compiler implementation.
 pub const AtomicOrder = enum {
+    /// The operation is atomic but does not have to have an observable modification order.
     Unordered,
+    /// Enforces an ordering of reads and writes on its value to other threads by having a modification order.
+    /// All memory accesses in program order are visible to the current thread.
```

Comment:
- "The operation is atomic but has a stable ordering of reads and writes to a given atomic variable (modification order)"
- The "program order" bit can be omitted as that's true for all Orderings, even non-atomic/normal memory operations.